ivadomed/main.py

```diff
@@ -286,6 +288,19 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
                                               eval_params=context["evaluation_parameters"])
         return df_results, pred_metrics
 
+    if command == 'segment':
+        bids_ds = bids.BIDS(context["loader_parameters"]["bids_path"])
+        bids_subjects = [s for s in bids_ds.get_subjects() if s.record["subject_id"] in test_lst]
+        for subject in bids_subjects:
+            fname_img = subject.record["absolute_path"]
+            pred = imed_utils.segment_volume(os.path.join(context['log_directory'], context['model_name']),
+                                             fname_image=fname_img,
+                                             gpu_number=context['gpu'])
```

Comment:
We should add here the post-processing params like done in SCT?

Reply:
they can still be applied [through the config file ](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/utils.py#L577), but we might want to modify the config file in packaged model.

Reply:
Oh! Yup! I was "we need to feed `options` for the postpro" but in this application we will use the postpro params defined in the config file! All good :-)